extension/test/unit/create-session-with-splits.handler.spec.js

```diff
@@ -75,80 +68,70 @@ describe('make-payment-with-splits::execute', () => {
-      )
-      expect(setMethodInfoMethod.method).to.equal('scheme')
-
-      const setMethodInfoName = response.actions.find(
-        (a) => a.action === 'setMethodInfoName'
-      )
-      expect(setMethodInfoName.name).to.eql({ en: 'Credit Card' })
+      expect(response.actions).to.have.lengthOf(3)
 
       const addInterfaceInteraction = response.actions.find(
         (a) => a.action === 'addInterfaceInteraction'
       )
-      expect(addInterfaceInteraction.fields.type).to.equal('makePayment')
+      expect(addInterfaceInteraction.fields.type).to.equal('createSession')
       expect(addInterfaceInteraction.fields.request).to.be.a('string')
       expect(addInterfaceInteraction.fields.response).to.be.a('string')
       expect(addInterfaceInteraction.fields.createdAt).to.be.a('string')
 
       const request = JSON.parse(addInterfaceInteraction.fields.request)
       const requestBody = JSON.parse(request.body)
       expect(requestBody.reference).to.deep.equal(
-        makePaymentWithSplitsRequest.reference
+        createSessiontWithSplitsRequest.reference
       )
       expect(requestBody.riskData).to.deep.equal(
-        makePaymentWithSplitsRequest.riskData
-      )
-      expect(requestBody.paymentMethod).to.deep.equal(
-        makePaymentWithSplitsRequest.paymentMethod
+        createSessiontWithSplitsRequest.riskData
       )
+
       expect(requestBody.browserInfo).to.deep.equal(
-        makePaymentWithSplitsRequest.browserInfo
+        createSessiontWithSplitsRequest.browserInfo
       )
       expect(requestBody.amount).to.deep.equal(
-        makePaymentWithSplitsRequest.amount
+        createSessiontWithSplitsRequest.amount
       )
       expect(requestBody.merchantAccount).to.equal(adyenMerchantAccount)
 
       const setCustomFieldAction = response.actions.find(
         (a) => a.action === 'setCustomField'
       )
-      expect(setCustomFieldAction.name).to.equal('makePaymentResponse')
+      expect(setCustomFieldAction.name).to.equal('createSessionResponse')
       expect(setCustomFieldAction.value).to.be.a('string')
       expect(setCustomFieldAction.value).to.equal(
         addInterfaceInteraction.fields.response
       )
 
       const setKeyAction = response.actions.find((a) => a.action === 'setKey')
-      expect(setKeyAction.key).to.equal(makePaymentWithSplitsRequest.reference)
-
-      const addTransaction = response.actions.find(
-        (a) => a.action === 'addTransaction'
-      )
-      expect(addTransaction.transaction).to.be.a('object')
-      expect(addTransaction.transaction.type).to.equal('Authorization')
-      expect(addTransaction.transaction.state).to.equal('Success')
-      expect(addTransaction.transaction.interactionId).to.equal(
-        JSON.parse(paymentSuccessResponse).pspReference
+      expect(setKeyAction.key).to.equal(
+        createSessiontWithSplitsRequest.reference
       )
+      // TODO : Checking if transaction is added after creating session
```

Comment:
As we already discussed about this topic, We don't need to add Transaction for session request. So you can remove this TODO.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1056/commits/254ca0ff56247f0a0be34c30162e4bb82200db3c

Reply:
submit-payment-details.handler.js
Since this is removed in handler, Can we remove the handler and tests.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1056/commits/aaa4c1791b53e18cab236dec27145616ed243ddf
https://github.com/commercetools/commercetools-adyen-integration/pull/1056/commits/ee3fc50f1529b3c7a673f6b0770e1cc650ecdf20

Reply:
I still see some redundant code after removing make-payment and submit-additional-details handlers.

1. Unit tests: 
ex: Payment-handler.spec.js(line 42)
Payment-handler-lineitem.spec.js.

It is better if we clean up completely.

Reply:
We still need `payment-handler-lineitem.spec.js`. It has been already calling /session endpoint.

I replaced `makePayment` and `submitAdditionalDetails` in `refund` and `manual` capture test cases as well as `validator`


validator
https://github.com/commercetools/commercetools-adyen-integration/pull/1056/commits/d8b8f0a061b7550ca2b444fa8daecd58f17c2c5e

refund unit test
https://github.com/commercetools/commercetools-adyen-integration/pull/1056/commits/b718c2cd7f8ffef4b10c5755276978ee587bb5ea

manual capture unit test
https://github.com/commercetools/commercetools-adyen-integration/pull/1056/commits/90de8fe7d9e87f11b7e346bc700eac816cccf51c

Reply:
We still need `payment-handler-lineitem.spec.js`. It has been already calling /session endpoint.

I replaced `makePayment` and `submitAdditionalDetails` in `refund` and `manual` capture test cases as well as `validator`


validator
https://github.com/commercetools/commercetools-adyen-integration/pull/1056/commits/d8b8f0a061b7550ca2b444fa8daecd58f17c2c5e

refund unit test
https://github.com/commercetools/commercetools-adyen-integration/pull/1056/commits/b718c2cd7f8ffef4b10c5755276978ee587bb5ea

manual capture unit test
https://github.com/commercetools/commercetools-adyen-integration/pull/1056/commits/90de8fe7d9e87f11b7e346bc700eac816cccf51c

Removed obsoleted unit test fixtures
https://github.com/commercetools/commercetools-adyen-integration/pull/1056/commits/a2c6e59206f10f51458aab6d0be65c720eb0d87f